src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -118,13 +118,13 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="SetColorSchemeParentCommandName" xml:space="preserve">
-    <value>Select color scheme</value>
+    <value>Change color scheme</value>
   </data>
   <data name="NewTabParentCommandName" xml:space="preserve">
-    <value>New tab</value>
+    <value>Open a new tab</value>
   </data>
   <data name="SplitPaneParentCommandName" xml:space="preserve">
-    <value>Split pane</value>
+    <value>Split a new pane</value>

```

Comment:
I think "Split pane" made more sense here, because you're not splitting the new pane, but rather the current one.

Reply:
agreed.